litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/styles.ts

```diff
@@ -6,6 +6,7 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.text.primary,
     paddingTop: theme.spacing(2),
     margin: '0 auto',
+    marginBottom: theme.spacing(-2), // Overriding because LitmusStepper has default padding bottom
```

Comment:
can we override it in the stepper or provide a prop to do so, rather than this negative margin?

Reply:
Actually, LitmusStepper uses a default padding top and bottom, changing that would affect all the stepper content and we only needed to override one single property for this screen only thus the negative margin.